src/plugins/intel_gna/optimizer/gna_pass_manager.cpp

```diff
@@ -978,6 +979,33 @@ void InsertCopyLayerPass::run() {
             }
         }
     }
+
+    for (auto & l : *pLayers) {
+        if (!l->insData.empty()) continue;
+
+        for (auto output : l->outData) {
+            auto& inputTo = getInputTo(output);
+            for (auto& childLayer : inputTo) {
```

Comment:
What if we have a case with only one Parameter layer? getInputTo(output) will be empty, so we'll do nothing here.

Reply:
As far as I understand, we still can't support one-layer nets, due to:

https://github.com/openvinotoolkit/openvino/blob/d520e5558fbeb0eb8c956e8a120b27b7b562e48c/src/plugins/intel_gna/backend/gna_limitations.cpp#L368

```
        if (secondLayers.empty()) {
            errMessage = "Network consists of input layer only (GNA)\n";
            return false;
        }
```

And other logic inside plugin

Reply:
theoretically I don't see any significant limitations to support them but the question is if it can be useful